tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -362,15 +369,20 @@ class MklDnnConvUtil {
     // Output batch is same as input batch.
     int out_batch = GetTensorDim(input_shape, data_format_, 'N');
     int out_depth;
-    
+
     // TODO add support for 3-D Depthwise
 
-    // Output depth is same as last dimension for filters for regular 
-    // convolutions. For depthwise it is in_depth*channel_muliplier 
+    // Output depth is same as last dimension for filters for regular
+    // convolutions.
+    // For depthwise it is in_depth * channel_multiplier. The channel_miltipler
```

Comment:
The comment line above this line is short. Could you please put this line with the previous line? (And wrap the line again if it's too long). Also, a new typo at the end of the line: `channel_miltipler` should be `channel_multiplier`.

Reply:
Not a problem, @penpornk. Thanks for reviewing.